_data/2018/capetown/faculty.yml

```diff
@@ -0,0 +1,17 @@
+Local faculty
```

Comment:
This is not valid YAML. See the [primary site file as an example](https://github.com/compsocialscience/summer-institute/blob/1d69e8edab5afa2a21ec45b8d94df69d5bcc5a07/_data/2018/faculty.yml).